web/ui/react-app/src/pages/graph/MetricsExplorer.tsx

```diff
@@ -1,33 +1,57 @@
-import React, { Component } from 'react';
-import { Modal, ModalBody, ModalHeader } from 'reactstrap';
+import React, { Component, ChangeEvent } from 'react';
+import { Modal, ModalBody, ModalHeader, Input } from 'reactstrap';
+import { Fuzzy, FuzzyResult } from '@nexucis/fuzzy';
+const fuz = new Fuzzy({ pre: '<strong>', post: '</strong>', shouldSort: true });
```

Comment:
Style nit: separate this line from the `import`s above with an empty line.

Reply:
should be fixed in last commit.